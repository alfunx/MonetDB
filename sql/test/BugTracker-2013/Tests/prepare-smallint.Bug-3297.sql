CREATE TABLE ints ( val int );
prepare INSERT INTO ints VALUES ( ? - 20 );
exec **(1000);
select * from ints;
drop table ints;
