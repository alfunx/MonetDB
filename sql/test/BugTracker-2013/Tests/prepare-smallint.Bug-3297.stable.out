stdout of test 'prepare-smallint.Bug-3297` in directory 'sql/test/BugTracker-2013` itself:


# 20:19:33 >  
# 20:19:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35817" "--set" "mapi_usock=/var/tmp/mtest-25917/.s.monetdb.35817" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/home/niels/scratch/rc-clean/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2013" "--set" "mal_listing=0"
# 20:19:33 >  

# MonetDB 5 server v11.15.4
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2013', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 3.777 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://niels.nesco.mine.nu:35817/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-25917/.s.monetdb.35817
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_history.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_compress.sql
# loading sql script: 18_dictionary.sql
# loading sql script: 19_cluster.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 24_zorder.sql
# loading sql script: 25_debug.sql
# loading sql script: 39_analytics.sql
# loading sql script: 40_geom.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_udf.sql
# loading sql script: 99_system.sql

# 20:19:34 >  
# 20:19:34 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-25917" "--port=35817"
# 20:19:34 >  

#CREATE TABLE ints ( val int );
#prepare INSERT INTO ints VALUES ( ? - 20 );
#prepare INSERT INTO ints VALUES ( ? - 20 );
% .prepare,	.prepare,	.prepare,	.prepare,	.prepare,	.prepare # table_name
% type,	digits,	scale,	schema,	table,	column # name
% varchar,	int,	int,	str,	str,	str # type
% 6,	2,	1,	0,	0,	0 # length
[ "bigint",	64,	0,	NULL,	NULL,	NULL	]
#exec  0(1000);
[ 1	]
#select * from ints;
% sys.ints # table_name
% val # name
% int # type
% 3 # length
[ 980	]
#drop table ints;

# 20:19:34 >  
# 20:19:34 >  "Done."
# 20:19:34 >  

